src/Domain/CoordinateMaps/EquatorialCompression.hpp

```diff
@@ -91,17 +97,20 @@ class EquatorialCompression {
  private:
   template <typename T>
   std::array<tt::remove_cvref_wrap_t<T>, 3> angular_distortion(
-      const std::array<T, 3>& coords, double inverse_alpha) const;
+      const std::array<T, 3>& coords, double inverse_alpha,
+      size_t index_pole_axis) const;
```

Comment:
Why do `angular_distortion` and `angular_distortion_jacobian` below take the `inverse_alpha` and `index_pole_axis` arguments? The functions are only used internally and the values passed to `inverse_alpha` and `index_pole_axis` are the member variables `aspect_ratio_` and `index_pole_axis_`, respectively.  Can't you just use the member variables within the functions rather than passing them?

Reply:
the coordinate map and the inverse coordinate map are the same, except for the value of `alpha` (the `alpha` for the inverse map is the inverse of the `alpha` for the forward map) so (inverse) `alpha` is left as an argument to `angular_distortion` (and `angular_distortion_jacobian`), which the (inverse) coordinate map and jacobian then call, where they provide the correct value of (inverse) `alpha` internally.

in the case of `index_pole_axis`, this is the same for both the forward map and inverse map, so you're right that I can just use the member variables instead, I will fix this!

Let me know if I misunderstood your question :+1:

Reply:
Ah yes I didn't see that the arguments were slightly different, thank you for pointing that out!